<launch>                                                                                         
  <node name="logger" pkg="f1tenth_course" type="logger" respawn="false" args="-b /home/amrl_user/bagfiles/" />
</launch>
